Packs/PhishingURL/Scripts/DBotPredictURLPhishing/DBotPredictURLPhishing.py

```diff
@@ -460,8 +460,15 @@ def extract_created_date(entry: dict):
 
 def weed_rasterize_errors(urls: list[str], res_rasterize: list[dict]):
     '''Remove the URLs that failed rasterization and return them.'''
-
-    error_idx = [i for (i, res) in enumerate(res_rasterize) if isinstance(res['Contents'], str)]
+    if len(urls) != len(res_rasterize):
+        demisto.debug(f'{res_rasterize=}')
+        raise DemistoException('Unexpected response from the "rasterize" command. '
+                               'Please make sure the Rasterize pack version is above 2.0.7')
+    error_idx = []
+    for (i, res) in enumerate(res_rasterize):
+        if isinstance(res['Contents'], str):
+            error_idx.append(i)
+    error_idx.reverse()
```

Comment:
you collect this into error_idx just to do a list comprehension on it later 
`[{'URL': urls.pop(i), 'Message': res_rasterize.pop(i)['Contents']} for i in error_idx]`
lets just do the whole thing in 1 loop. why `error_idx.reverse()`?

Reply:
Because we need to remove elements and we can't do that as we're iterating over the list.

Reply:
I'll put the index list back into a list comprehension as that was a misunderstanding of your intention.